results.html

```
@@ -0,0 +1,734 @@
+    <div class="verdicts" id="leverList"></div>
+    <p class="note">Each verdict is a real sweep with its own pre-registered rule. "Backfired" and "wash" are genuine negative results — kept on the record, not quietly dropped.</p>
+  </div>
+</section>
+
+<!-- 06 resolution -->
+<section class="reveal">
+  <div class="wrap">
+    <div class="shead"><span class="snum">06 / 10</span><span class="skick">The resolution</span><span class="bar"></span></div>
+    <h2><span class="hicon"><svg class="ic" aria-hidden="true" focusable="false"><use href="#trend"/></svg></span>The failure wasn't the design — it was the dose</h2>
+    <p class="lead">When a third resident joins, each agent gets the spotlight less often, so the same cost spread over more residents pushes each one less. The fix wasn't a cleverer design; it was simply <strong>matching the dose to the bigger village</strong> (raising the contribute cost from 30 to 42). Do that, and the three-resident village clears the line: divergence rises from <strong>0.21 to 0.31</strong>, passing on all three fresh seeds.<a class="cite" href="https://github.com/TheIllusionOfLife/microverse/blob/main/docs/adr/0018-action-economy-r3-levers2.md" target="_blank" rel="noopener">ADR&nbsp;0018</a></p>
+    <div class="callout win"><b>An honest bound:</b> this is <em>one rescued three-resident village</em>, not a proven law. It's good evidence for a simple "dilution" explanation — bigger village needs a bigger dose — and it cleared the differentiation bar on all three seeds and the stricter role-stability check on two of three. It is <em>not</em> yet shown to hold for much larger populations, other action menus, or other models.</p></div>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </p> ]

(tag-pair)